### PR TITLE
Design logo and add favicon to page

### LINKED ISSUE #31: Replace favicon for something that isn't the default one
Probably the SS14 logo or something.

### PR DESCRIPTION
**Pull request by bhenrich**
Designed a simple logo on theme with replays and the ss14 logo. Added as favicons and PNGs.

Fixes #31

### CODE REVIEW

**Comment by bhenrich** [comment]
don't mind all the weird commits i accidentally did all this on master

**Comment by Simyon264** [comment]
> don't mind all the weird commits i accidentally did all this on master

its fine dw

File: ReplayBrowser/appsettings.json [hunk 1]
```diff
@@ -17,23 +17,23 @@
     },
```
Comment (Simyon264) [inline ReplayBrowser/appsettings.json:17]: Please revert the changes in appsettings.json, the current deploy script to not overwrite this, so this results in no downloads being done.

**Review by Simyon264 — CHANGES_REQUESTED** [review]

File: ReplayBrowser/appsettings.json (hunk 1 above)
Comment (bhenrich) [reply ReplayBrowser/appsettings.json:17]: i fucking triple checked that i didnt push that, what the fuck

Comment (Simyon264) [reply ReplayBrowser/appsettings.json:17]: should have checked 4 times :godo:

**Comment by Simyon264** [comment]
coolio
